src/app/lib/business/logger/logger_manager.dart

```diff
@@ -62,7 +66,7 @@ final class _LoggerManager implements LoggerManager {
   @override
   bool hasConfigurationBeenChanged = false;
 
-  _LoggerManager(this._loggerRepository);
+  _LoggerManager(this._loggerRepository, this._alice);
```

Comment:
Probably should use the other way of setting them, normally we don't expose private member with 'this'.